id/proguard-rules.pro

```diff
@@ -58,8 +58,8 @@
 #net.zetetic:android-database-sqlcipher
 -keep class net.sqlcipher.** { *; }
 
-# Do not obfuscate names in out packages`
--keepnames class com.simprints.** { *; }
+# Do not obfuscate names in simprints package
+-keep class com.simprints.** { *; }
```

Comment:
Is this a related change?

Reply:
yes 
As seen in this [trail](https://github.com/Simprints/Android-Simprints-ID/actions/runs/9974659574/job/27562603862), we should keep the class functions and fields, not just the names  
 https://github.com/Simprints/Android-Simprints-ID/actions/runs/9974659574/job/27562603862